lib/http-context.js

```diff
@@ -384,6 +424,11 @@ HttpContext.prototype.done = function() {
   var data = this.result;
   var res = this.res;
   var accepts = this.req.accepts(this.supportedTypes);
+  var defaultStatus = this.method.http.status;
+
+  if (defaultStatus) {
+    res.status(defaultStatus);
```

Comment:
Should we check that `res.statusCode === undefined || res.statusCode === 200` before applying `defaultStatus`?